client/layout/index.jsx

```diff
@@ -179,7 +178,7 @@ export default connect(
 		const { isLoading, section, hasSidebar, isFullScreen, chunkName } = state.ui;
 		return {
 			isLoading,
-			isSupportUser: isSupportUser( state ),
+			isSupportUser: state.support.isSupportUser,
```

Comment:
Previously this selector was using the support user and token in the Redux store to determine whether support user was active. Now that we're storing those outside the Redux store, I've opted for just a boolean flag in the Redux state to keep track of whether support user is active or not. We could keep the selector, but it would just be:

```js
const isSupportUser = ( state ) => state.support.isSupportUser
```